packages/form-plugin/src/actions.ts

```diff
@@ -3,7 +3,12 @@ export class UpdateFormStatus {
     // NOTE: Not necessary to declare the type in this way in your code. See https://github.com/ngxs/store/pull/644#issuecomment-436003138
     return '[Forms] Update Form Status';
   }
-  constructor(public payload: { status: string | null; path: string }) {}
+  constructor(
+    public payload: {
+      status: 'VALID' | 'INVALID' | 'PENDING' | 'DISABLED' | null;
```

Comment:
maybe use enum for status?

```ts
enum UpdateStatusType {
  VALID = "VALID"
  INVALID = "INVALID",
  ...
}
```

Reply:
@splincode yeah, I thought about it, the problem is that it can be a breaking change for our users.

P.S. `@angular/forms` doesn't use `enum`.

Reply:
Okay, use type alias for our code

```
type UpdateFormStatus = 'VALID' | 'INVALID' | 'PENDING' | 'DISABLED' | null;
```

Reply:
@splincode done